.github/workflows/auto-release.yml

```diff
@@ -1,34 +1,92 @@
-# For @loganthomas!
-# Some hints for you :)
+# Good example resources
+# https://riggaroo.dev/using-github-actions-to-automate-our-release-process/
+# https://blog.eizinger.io/12274/using-github-actions-to-automate-gitflow-style-releases
 
-# Give it an informative name.
-name: Some not so informative name
+name: Auto-release
 
-# Decide what git events trigger this workflow.
-# One of the two entries below shouldn't be used (logically) for a release workflow.
 on:
   push: [master]
   pull_request: [master]
 
-# Define the jobs
+  # Decide what the new version number will be
```

Comment:
I *think* this allows us to explicitly input a version number for the release, but I'm not sure